buildenv/jenkins/common/build.groovy

```diff
@@ -500,6 +500,16 @@ def fetchFile(src, dest) {
 def archive_diagnostics() {
     def datestamp = variableFile.get_date()
     def diagnosticsFilename = "${JOB_NAME}-${BUILD_NUMBER}-${datestamp}-diagnostics.tar.gz"
+
+    def archiveCmd = "tar -zcvf ${diagnosticsFilename} -T -"
+
+    def findNameCore = "-name 'core.*.dmp'"
+    def findCmd = "find ."
+    findCmd += " ${findNameCore}"
+    findCmd += " -o -name 'javacore.*.txt'"
+    findCmd += " -o -name 'Snap.*.trc'"
+    findCmd += " -o -name 'jitdump.*.dmp'"
```

Comment:
Can you please sort the additions alphabetically, here and below?

Reply:
Sorted these and the debug info globs. I think the lines adding DDR-related stuff are already sorted well enough